components/developer/rust/pkg5

```diff
@@ -16,4 +16,4 @@
         "developer/lang/rustc"
     ],
     "name": "rustc"
-}
\ No newline at end of file
+}
```

Comment:
You added developer/clang-90 to the required packages. That should also have an impact on pkg5. You can either run gmake "update-metadata" or "gmake publish" to get an updated pkg5 file.

Reply:
gmake update-metadata ...
That's was easy.